packages/node-cli/src/core/sdksetup/SDKProperties.js

```diff
@@ -23,6 +23,10 @@ class SDKProperties {
 		return configFile.sdkDownloadUrl;
 	}
 
+	getSDKVersion() {
+		return SDK_VERSION;
+	}
+
 	getSDKFileName() {
 		return this.configFileExists() ? CONFIG_FILE_CACHE.sdkFilename : SDK_FILENAME;
```

Comment:
nit: isn't this a problem that `getSDKFileName` can be based on config file? the errors will be on the missing `SDK_VERSION`, no?

Reply:
You are right but I try to make as least changes as I can and this case is not really in our flow. Do you think that I should take care for it?

Reply:
I think a simple solution will be to use `getSDKFileName()` when building the exception

Reply:
But then the excepction will include cli-2020.1.0.jar in it's output instead of just the 2020.1.0 version.

Reply:
so maybe add log line before throwing the exception with `getSDKFileName`?